### DISCUSSION TITLE
icub-head is not responding

### DISCUSSION THREAD

**Discussion by justdiandes** [robotology/community#711] [Q&A] [answered]
Hi again! 

I was doing some setups related to [this repository](https://icub-tech-iit.github.io/documentation/icub_operating_systems/other-machines/icub-server-laptop/?h=ssh#install-the-ssh-keys-for-password-less-login-on-icub-head) for connecting my PC to the icub, but for some reason, after the configurations related to the keys, the icub-head stops working on every PC we had. Before, we had 2 PC's that could connect but now I have none. 

Using the yarpmanager we had this:
![Captura de pantalla de 2024-07-08 18-13-17](https://github.com/robotology/community/assets/121194422/a4e538d0-6d13-4760-95df-e1dec6d01dc3)

And the `yarp name list` only sees the icubsrv:
![Captura de pantalla de 2024-07-08 18-13-53](https://github.com/robotology/community/assets/121194422/84bf723b-32c0-4dc2-9c95-6e28c422c853)

The older issues that had relationship with the icub-head functioning were related to the configuration of the bashrc and that kind of stuff, but mine is looking really weird and I'm a little worried that some important file was edited. 
I still have the connection to icub-head via ssh, but the yarpmanager doesn't.

**Comment by Nicogene** [comment, marked as answer]
Hi @justdiandes,

Have you checked that the yarpserver configuration of `icub-head` is the same of the laptop?

You should do
```
$ yarp conf
/path/to/.config/yarp/yarp.conf
```

and then
```
cat /path/to/.config/yarp/yarp.conf
```

It contains IP and port where the yarpserver should run, what is contained in `yarp.conf` should match in `icub-head` and the laptop in order to communicate

**Reply by justdiandes** [reply]
Yes! 
yesterday we could figure out what's wrong in the settings and it was that file. I don't know yet how this file was overwritten but now it works fine.

Thanks!
